gamestonk_terminal/options/README.md

```diff
@@ -84,6 +84,17 @@ Display options chains. [Source: Tradier]
 * --puts : Flag to show puts only
 * -m : Minimum strike price to consider.
 * -M : Maximum strike price to consider.
+* -d       : Columns to display.  Should be comma separated.  Must be in `['bid', 'ask', 'strike', 'bidsize', 'asksize', 'volume', 'open_interest',
+        'delta', 'gamma', 'theta', 'vega', 'ask_iv', 'bid_iv','mid_iv']`
 
 
 <img width="948" alt="chains" src="https://user-images.githubusercontent.com/25267873/115161876-f708ff80-a097-11eb-8073-195979862a45.png">
+
+To change what columns are displayed, one can use the `-d` flag.  To show the same as above, but only the ask, ask size 
```

Comment:
I don't think we need this information to follow remaining README.

I think the idea is good, but if we were to add all different options for all different commands, I think these would become quite big and the idea is that the user scrolls through and gets an idea of what he can from the terminal.